src/aomusicplayer.cpp

```diff
@@ -32,6 +32,11 @@ void AOMusicPlayer::play(QString p_song, int channel, bool loop,
   else
     newstream = BASS_StreamCreateFile(FALSE, f_path.utf16(), 0, 0, flags);
 
+  if (f_path.startsWith("http"))
+    newstream = BASS_StreamCreateURL(f_path.toStdWString().c_str(), 0, BASS_STREAM_AUTOFREE | BASS_UNICODE | BASS_ASYNCFILE, NULL, NULL);
```

Comment:
```suggestion
    newstream = BASS_StreamCreateURL(f_path.utf16(), 0, BASS_STREAM_AUTOFREE | BASS_UNICODE | BASS_ASYNCFILE, NULL, NULL);
```

Reply:
doing that causes an invalid conversion error that can only be overridden by an -fpermissive flag in windows. Still broken on linux and macosx.

Reply:
just like @stonedDiscord said. The bass header file doesn't allow streaming to work on other platforms even though the library itself has the symbol definitions to support it.